Install-Under-CentOS_LAMP.md

```diff
@@ -401,5 +408,12 @@ documentation on how to make your SELinux policy right.
 **Note:** If you installed Cacti out of `/var/www/html` make sure you fix up
 all SELinux context and permissions.
 
+### Use setup wizard script for an interactive installation
```

Comment:
Whilst the script is useful for people to try out, I'm not sure it should be included in the official documentation.

Reply:
This link has now been removed so can be considered resolved